classes/maps.php

```diff
@@ -285,8 +280,7 @@ function addMapMarkers() {
 
 				</script>
 
-				<div id="map_canvas"
-				     style="height: <?php echo esc_attr( $_GET['map_height'] ); ?>; width: <?php echo esc_attr( $_GET['map_width'] ); ?>; border: 1px solid #eee; overflow: hidden"></div>
+				<div id="map_canvas" style="height: 100%; width: 100%; border: 1px solid #eee; overflow: hidden"></div>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->